docs/help/specifications-support/markdown-support.md

```diff
@@ -62,6 +62,72 @@ will render:
 this is an important information to **standout**
 :::
 
+## Images
+
+Use the following syntax to add images in your markdown
+```
+![Alt text](/path/to/image.jpg "Image title")
+```
+
+Note that even it's a best practice to always add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, the parameters `Alt text` and `"Image title"` are optional.
```

Comment:
I don't understand the beginning of the sentence, I think you mixed two sentences :slightly_smiling_face:, can you check please?

Reply:
Oh I got it there's a `if` missing:

```suggestion
Note that even if it's a best practice to always add an [alt text](https://en.wikipedia.org/wiki/Alt_attribute#Usage) for an image, the parameters `Alt text` and `"Image title"` are optional.
```

Reply:
it's valid english even without the if but i'll add it for clarity :) Thank you for the heads up

Reply:
Oh! Sorry I didn't know, thanks 🙂